api/build.sh

```diff
@@ -20,6 +20,7 @@ export ENV=local
 pwd=`pwd`
 
 VERSION=$(cat ./api/VERSION)
+GITHASH=$(git log --pretty=format:"%h" -1)
```

Comment:
oh yes, if a user is using a release tar package from Apache Release, the package will not include `.git` folder.

cc @membphis any ideas?

Reply:
For the version released by apache, we add a new separate file to store the Git hash value.

Here is an example:

```
$ cat ./.githash
bae2e05b
```

Reply:
@nic-chen what do you think?

Reply:
and I think we generate the file automatically:

https://github.com/apache/apisix-dashboard/blob/master/Makefile#L96

Reply:
It seems ingress apisix also has this issue.

Reply:
> For the version released by apache, we add a new separate file to store the Git hash value.
> 
> Here is an example:
> 
> ```
> $ cat ./.githash
> bae2e05b
> ```

LGTM

Reply:
@membphis @juzhiyuan @nic-chen @tokers

Thank you all for reviewing and suggestions. How about try to get the git hash from `./.githash` first, if that file does not exist, we use the `.git` folder?

Reply:
@membphis @juzhiyuan @tokers @nic-chen 

Much appreciated for your suggestions. PR already is updated and adds support for the new `.githash` file. Ready for review. Thanks.